source/jormungandr/jormungandr/stat_manager.py

```diff
@@ -50,6 +50,10 @@
 f_datetime = "%Y%m%dT%H%M%S"
 
 
+class StatManagerError(RuntimeError):
+    pass
```

Comment:
This should be moved in exceptions.py, no?

Reply:
Done